### DISCUSSION TITLE
Sudo plugin

### DISCUSSION THREAD

**Discussion by noahfraiture** [nushell/nushell#14139] [Q&A]
Hello,

On ZSH you can have OMZ plugins such as `sudo` to add a `sudo` at the beginning of your command with twice esc. Is it something possible with nushell ? I'm using vi mode, IDK if it's relevant

**Comment by realth000** [comment]
I'm using this shortcut to prepend `sudo ` by pressing `alt` + `/`.

In vi mode you can do something similar I think.

Add the following code to keybindings section in `config.nu` where already exists lots of shortcuts.

```nu
        {
            name: prepend_sudo
            modifier: alt
            keycode: char_/
            mode: emacs
            event: [
              { edit: MoveToStart }
              { edit: InsertString, value: "sudo " }
              { edit: MoveToEnd }
            ]
        }
```

This shortcut will move cursor to the start of the line, insert `sudo ` and then move cursor to the end of line, covers most of the usage， in emacs mode.

For more keybinding infos, use the `keybindings list` command to check all available keys and events.
